### PR TITLE
#424 Allow document preview before sending a message and update antd

### PR DESCRIPTION
**Pull request by stephanevieira75**
### Preview
https://user-images.githubusercontent.com/36481167/123407634-75a87e80-d5ac-11eb-8d6c-926dbd8364fd.mp4

### CODE REVIEW

**Review by chamerling — APPROVED** [review]

**Comment by RomaricMourgues** [comment]
@stephanevieira75 could you include the Snyk antd version update ?

File: twake/frontend/src/app/scenes/Apps/Drive/Viewer/Viewer.js [hunk 1]
```diff
@@ -103,7 +103,7 @@ export default class Viewer extends Component {
           <div className="view_header">
             <div className="title">{current.name}</div>
 
-            {editor_candidate.length == 1 && (
+            {DriveService.previewonly === false && editor_candidate.length == 1 && (
```
Comment (chamerling) [inline twake/frontend/src/app/scenes/Apps/Drive/Viewer/Viewer.js:106]: `!DriveService.previewonly` should be better than comparing to `false`

File: twake/frontend/src/app/scenes/Apps/Drive/Viewer/Viewer.js [hunk 2]
```diff
@@ -138,20 +138,22 @@ export default class Viewer extends Component {
...
-                  window.open(link, '_blank');
-                } else {
-                  window.open(link);
-                }
-              }}
-            >
-              {current.url && <OpenInNewIcon class="m-icon-small" />}
-              {!current.url && <DownloadIcon class="m-icon-small" />}
-            </div>
+            {DriveService.previewonly === false && (
```
Comment (chamerling) [inline twake/frontend/src/app/scenes/Apps/Drive/Viewer/Viewer.js:141]: Also here

File: twake/frontend/src/app/scenes/Apps/Drive/Viewer/Viewer.js [hunk 3]
```diff
@@ -138,20 +138,22 @@ export default class Viewer extends Component {
...
+              <div
+                className="download"
+                onClick={() => {
+                  var link = DriveService.getLink(current, null, true);
+                  if (current.url) {
+                    window.open(link, '_blank');
+                  } else {
+                    window.open(link);
```
Comment (chamerling) [inline twake/frontend/src/app/scenes/Apps/Drive/Viewer/Viewer.js:149]: Use a ternary

File: twake/frontend/src/app/scenes/Apps/Drive/Viewer/Viewer.js [hunk 4]
```diff
@@ -115,7 +115,7 @@ export default class Viewer extends Component {
                 )}
               </Button>
             )}
-            {editor_candidate.length > 1 && (
+            {DriveService.previewonly === false && editor_candidate.length > 1 && (
```
Comment (chamerling) [inline twake/frontend/src/app/scenes/Apps/Drive/Viewer/Viewer.js:118]: Also here

**Review by chamerling — CHANGES_REQUESTED** [review]

File: twake/frontend/src/app/scenes/Apps/Drive/Viewer/Viewer.js (hunk 1 above)
Comment (stephanevieira75) [reply twake/frontend/src/app/scenes/Apps/Drive/Viewer/Viewer.js]: Done

File: twake/frontend/src/app/scenes/Apps/Drive/Viewer/Viewer.js (hunk 3 above)
Comment (stephanevieira75) [reply twake/frontend/src/app/scenes/Apps/Drive/Viewer/Viewer.js]: Done 👍🏻

**Review by chamerling — APPROVED** [review]

**Pull request merged**